### DISCUSSION TITLE
Spawn Enemies on Player & Spawn items (including turrets and landmine)

### DISCUSSION THREAD

**Discussion by ivjvon** [winstxnhdw/lc-hax#279] [Ideas]
Spawn different enemies on player

can be 

random
baboonhawk
forestgiant
ghostgirl
flowerman
thumper
spider
nutcracker
snareflea
mask
jester
blob
earthleviathan (worm)

Spawn items (all of the items including a turret and a landmine)

**Comment by winstxnhdw** [comment]
If I knew these were possible, I would've implemented them ages ago.

**Reply by ivjvon** [reply]
how come some of the hax had the option to spawn a turret and a landmine, and another one that  spawns random monsters to a player, just asking.

**Reply by winstxnhdw** [reply]
Which? Can you send a link? I am curious too.

**Reply by joep26020** [reply]
Yeah, send the other hack

**Reply by o777l** [reply]
I mean there is Game Master as a mod in Thunderstore but requires host. Then there is Lethal Menu wich can spawn mines without host and supposedly turrets too. It says spawn mimic but did not work without host.

**Reply by winstxnhdw** [reply]
LethalMenu says spawn landmine is host-only. Most features in other cheats are mostly copy-pastes of this project so I’d be surprise to see a feature on another project that isn’t already in here.
